### ISSUE TITLE
Improve PayoutsRenameNotice code and add more tests.

### ISSUE THREAD

**Issue by shendy-a8c** [Automattic/woocommerce-payments#9692]
This issue is a follow-up from https://github.com/Automattic/woocommerce-payments/pull/9680#issuecomment-2466584119.

We implemented PayoutsRenameNotice and need to merge https://github.com/Automattic/woocommerce-payments/pull/9680 to make it into the next release. However, there are couple things that might need to be improved:
- The use of `useEffect()` as pointed by https://github.com/Automattic/woocommerce-payments/pull/9680#discussion_r1835575486.
- Need more unit tests.

**Comment by shendy-a8c** [comment]
I move this issue out of payout rename project as it is more of a maintenance issue.

**Comment by haszari** [comment]
> The use of useEffect() as pointed by https://github.com/Automattic/woocommerce-payments/pull/9680#discussion_r1835575486.
Need more unit tests.

@shendy-a8c  is it worth keeping this issue open, since this notice will be removed before too long. This issue is quite vague, so it's unlikely to get actioned. If there's a problem here that's important to solve, please log an issue for that, and feel free to close this.

- https://github.com/Automattic/woocommerce-payments/issues/9704

**Comment by shendy-a8c** [comment]
> @shendy-a8c is it worth keeping this issue open, since this notice will be removed before too long.

Nope. This issue can be closed. We have removed the notice https://github.com/Automattic/woocommerce-payments/issues/9704

Sorry I missed your comment.
